### PR TITLE
A document on upgrading cassandra.

### PR DESCRIPTION
**Pull request by porridge**
Baby step towards https://github.com/kudobuilder/kudo.dev/issues/149

Note the sample output assumes that https://github.com/kudobuilder/kudo/issues/1279 is already fixed.

### CODE REVIEW

File: docs/upgrading.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+# Upgrade KUDO Cassandra
+
+This runbook explains how to upgrade a running KUDO Cassandra instance to a newer version of KUDO Cassandra.
```
Comment (mpereira) [inline docs/upgrading.md]:
```suggestion
This guide explains how to upgrade a running KUDO Cassandra instance to a newer version of KUDO Cassandra.
```

File: docs/upgrading.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+namespace_name=default
+destination_version=0.1.2
+```
+
+#### 2. Verify that the variables are set correctly
+
+```bash
+kubectl get instance $instance_name -n $namespace_name
+echo About to upgrade to $destination_version
+
```
Comment (mpereira) [inline docs/upgrading.md]:
```suggestion
```

File: docs/upgrading.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+Plan(s) for "cassandra" in namespace "default":
+.
+└── cassandra (Operator-Version: "cassandra-0.1.2" Active-Plan: "deploy")
+    └── Plan deploy (serial strategy) [IN_PROGRESS]
+        └── Phase nodes (parallel strategy) [IN_PROGRESS]
+            └── Step node [IN_PROGRESS]
+
+```
+
+Once the pods are ready (passing readiness and liveness checks),the plan should change to `COMPLETE`.
```
Comment (mpereira) [inline docs/upgrading.md]:
```suggestion
Once the pods are ready (passing readiness and liveness checks), the plan should change to `COMPLETE`.
```

**Comment by porridge** [comment]
```
./tools/format_files.sh: line 33: prettier: command not found
```
:sob:

**Comment by porridge** [comment]
https://prettier.io/docs/en/install.html
```
$ yarn
bash: yarn: command not found
$ npm
bash: npm: command not found
```
:sob: :sob: :sob:

**Comment by mpereira** [comment]
Yeah going forward there should be a development Docker image for that. The dependencies are linked from the documentation fwiw... https://github.com/mesosphere/kudo-cassandra-operator/blob/master/DEVELOPMENT.md#additional-requirements

**Comment by mpereira** [comment]
https://github.com/mesosphere/kudo-cassandra-operator/pull/32/commits/4d528ef19a30ed0c858489aef9ff2d3f1778cef7

**Comment by porridge** [comment]
Managed to get it running using a docker container.

**Comment by porridge** [comment]
Suggestions applied, PTAL

**Comment by mpereira** [comment]
🚢

**Pull request merged**